app/src/main/res/layout/searchbar.xml

```diff
@@ -27,21 +28,32 @@
     android:focusableInTouchMode="true">
 
     <CheckBox
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
         android:id="@+id/onlyOfficialObjectsCheckbox"
+        android:layout_width="0dp"
+        android:layout_height="48dp"
+        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintTop_toTopOf="parent">
-    </CheckBox>
-
-    <EditText
-        android:id="@+id/searchEditText"
-        android:layout_width="wrap_content"
-        android:layout_height="45dp"
-        android:layout_marginTop="8dp"
-        android:maxLines="1"
-        android:singleLine="true"
-        app:layout_constraintRight_toRightOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    <com.google.android.material.textfield.TextInputLayout
```

Comment:
Maybe it makes sense for `PartnersActivity` to include this searchbar and make the checkbox above invisible at the activity's start-up instead of defining the same component on its own? Doing so, we could reduce quite some code-duplication.

Reply:
I could even think of re-using the checkbox for PartnerActivity e.g. for sth. like "sort by name" to either sort partners by name or number of shared routes.

Reply:
I thought about this, too but I didn't want to change too many things. Anyway, I will include this change into the the PR.